src/selection.coffee

```diff
@@ -404,6 +404,28 @@ class Selection extends Model
     @selectLeft() if @isEmpty() and not @editor.isFoldedAtScreenRow(@cursor.getScreenRow())
     @deleteSelectedText()
 
+  # Deprecated: Use {::deleteToBeginningOfWord} instead.
+  backspaceToBeginningOfWord: ->
```

Comment:
Looks like this might have been a merge gone wrong, these are at the bottom now in the deprecated block so I think they can just be removed completely from this diff.